projects/isy-angular-widgets/src/lib/hauptfenster/hauptfenster.component.html

```diff
@@ -42,73 +45,81 @@
     </ng-template>
   </p-megaMenu>
   <div class="isy-hauptfenster-titelzeile p-2" [style]="'background-color:' + applicationGroupColor + ';'">
-    <div *ngIf="title; else customTitelzeile">{{ title }}</div>
-    <ng-template #customTitelzeile>
+    @if (title) {
+      <div>{{ title }}</div>
+    } @else {
       <ng-content select="[Titelzeile]"></ng-content>
-    </ng-template>
+    }
   </div>
 
   <ng-content select="[Seitentoolbar]" class="dont-print"></ng-content>
 
   <div class="grid grid-nogutter flex-grow-1">
-    <div
-      class="isy-hauptfenster-linksnavigation dont-print"
-      *ngIf="showLinksnavigation"
-      [style.width]="collapsedLinksnavigation ? null : linksNavigationWidth"
-    >
-      <div [hidden]="collapsedLinksnavigation" id="open-links-navigation">
-        <div *ngIf="allowSidebarCollapse" class="collapseButton">
+    @if (showLinksnavigation) {
+      <div
+        class="isy-hauptfenster-linksnavigation dont-print"
+        [style.width]="collapsedLinksnavigation ? null : linksNavigationWidth"
+      >
+        <div [hidden]="collapsedLinksnavigation" id="open-links-navigation">
+          @if (allowSidebarCollapse) {
+            <div class="collapseButton">
+              <button
+                *ngIf="allowSidebarCollapse"
```

Comment:
The if instruction inside the button no longer seems to be necessary

Reply:
Removed unnecessary *ngif